internal/checkout/testdata/event.json

```diff
@@ -2,7 +2,7 @@
     "id": "github.com|00ea9422-5ec9-11ee-9135-d356b0e26c1e",
     "type": "PUSH",
     "provider": "github",
-    "providerURL": "https://github.com/fakeorg/fakerepo",
+    "providerURL": "https://github.com",
     "repositoryUrl": "https://github.com/fakeorg/fakerepo.git",
```

Comment:
The existing test data was wrong in this regard - fixed now.

(I already applied the same change to the DSL engine test data after confirming that in practice the `providerURL` field really only holds the server URL.)